backend/app/usecase/shortlink/creator.go

```diff
@@ -75,34 +75,37 @@ func (c CreatorPersist) CreateShortLink(shortLink entity.ShortLink, customAlias
 		return entity.ShortLink{}, ErrMaliciousLongLink(longLink)
 	}
 
-	if !c.isAliasProvided(customAlias) {
-		return c.createShortLinkWithAutoAlias(shortLink, user)
-	}
-
+	customAlias := shortLinkInput.GetCustomAlias("")
 	isValid, violation = c.aliasValidator.IsValid(customAlias)
```

Comment:
IsValid is accepting a pointer. We should pass in the pointer directly.

Reply:
Didn't see this comment earlier. As mentioned in the other comment, `validator` now accepts value rather than pointer. Should I change the interface back to accepting pointer?